content/departments/product-engineering/product/process/product_feedback_rotation.md

```diff
@@ -0,0 +1,88 @@
+# Product feedback rotation
+
+Sourcegraph collects user and stakeholder feedback from [several sources](user_stakeholder_feedback.md), each with an owner. Product managers are ultimately responsible for feedback in their areas. The product team as a whole is responsible as the first responder for replying and triaging feedback from some sources. To make sure this gets done, a given PM is the dedicated PM on product feedback rotation for rotations of two weeks.
+
+If you are the PM on rotation, your are responsible for:
+
+- [replying to relevant #feedback channel items](#triaging-and-answering-the-feedback-channel)
+- [labelling GitHub issues with no team label](#labelling-gitHub-issues-with-no-team-label)
+- [triaging productboard feedback](#triaging-producboard-feedback)
+
+Other teams are responsible for support issues (customer support) and product gaps (CEs).
+
+## Current rotation
+
+The current PM on rotation is listed [here](https://docs.google.com/document/d/1TTRjK-CL38fdCvrVUgRL70agUiwDbQFJXCo8IuJmLls/edit#heading=h.z3gywz4ddkbk).
+
+## End of a rotation
+
+At the end of a rotation, the PM on rotation is responsible for handing off the rotation to the next PM.
+
+## Feedback rotation guide
+
+### Triaging and answering the feedback channel
+
+When you're on support rotation, you are explicitly responsible for monitoring the #feedback channel. There are two things you need to do there:
+
+- answering some NPS submissions
+- routing feedback submitted directly by teammates to the right team
+
+There is a 24h SLA for this.
```

Comment:
Should we clarify this is only expected on weekdays? Or is it implicit?

Reply:
I'd clarify! Because on-call usually includes weekends. 
```suggestion
There is a 24h SLA for this on business days.
```